dev/components/graph.js

```diff
@@ -0,0 +1,117 @@
+import Chart from 'chart.js';
+
+/*
+**  GRAPH
+**
+*/
+
+class Graph {
+  // @parameters (string, array, array, string, object)
+  constructor(canvasId, newData, newLabels, graphType, options) {
+    this.canvasId = canvasId;
+    this.graphType = graphType || 'line';
+
+    this.data =
+      {
+        labels: newLabels,
+        display: true,
+        datasets: [{
+          backgroundColor: 'rgba(250, 128, 114, .2)',
+          borderColor: '#FA8072',
+          borderWidth: 2,
+          data: newData,
+          hoverRadius: 12,
+          pointBackgroundColor: 'rgba(250, 128, 114, 1)',
+          pointBorderColor: '#fff',
+          pointBorderWidth: 2,
+          pointHoverBorderColor: '#fff',
+          pointHoverBackgroundColor: '#000',
+          pointHoverRadius: 5,
+          pointRadius: 5,
+          radius: 4,
+          lineTension: .4
+        }]
+      };
+
+    this.options = options || this.getOptions();
+    this.graph;
+
+    this.renderGraph();
+  }
+
+
+  // DESTROY CHART
+  destroy() {
+    if (this.graph) {
+      this.graph.destroy();
+    }
+  }
+
+
+  // RENDER OPTIONS OBJECT
+  getOptions() {
+    return {
+      responsive: false,
+      layout: {
+        padding: {
+          top: 30,
+          right: 20,
+          bottom: 30,
+          left: 20
+        }
+      },
+      legend: {
+        display: false,
+        labels: {
+          // This more specific font property overrides the global property
+          fontColor: 'black',
+          fontFamily: "'Montserrat', sans-serif",
+        }
+      },
+      scales: {
+        xAxes: [{
+          gridLines : {
+            color: 'rgba(0,0,0,0.03)',
+            display : true,
+            tickMarkLength: 10
+          },
+          ticks: {
+            fontColor: '#B0BEC5',
+            fontFamily: "'Montserrat', sans-serif",
+            fontStyle: 'normal',
+            autoSkip: false
+          }
+        }],
+        yAxes: [{
+          gridLines: {
+            color: 'rgba(0,0,0,0.03)',
+            drawBorder: false,
+            zeroLineColor: 'rgba(0,0,0,0.04)',
+            tickMarkLength: 0
+          },
+          ticks: {
+            beginAtZero: false,
+            // stepSize: 1,
+            fontColor: '#B0BEC5',
+            fontFamily: "'Montserrat', sans-serif",
+            fontStyle: 'normal',
+            // min: 0,
+            padding: 15
+          }
+        }]
+      }
+    }
+  }
+
+
+  // RENDER NEW CHART
+  renderGraph() {
+    this.graph = new Chart(this.canvasId, {
+      type: this.graphType,
+      data: this.data,
+      options: this.getOptions()
```

Comment:
should this be `this.options`?  since in the constructor you checked for passed in options or default back to getOptions.